templates/project/project.yml

```diff
@@ -13,6 +13,9 @@ Parameters:
   ProjectDomainName:
     Type: String
     Default: ""
+  ProjectName:
+    Type: String
+    Default: ""
```

Comment:
Should `ProjectName` be required instead of optional?

Reply:
Good call.